src/test/java/sk/uniba/fmph/dcs/player_board/PlayerFiguresTest.java

```diff
@@ -0,0 +1,98 @@
+package sk.uniba.fmph.dcs.player_board;
+
+import org.json.JSONObject;
+import org.junit.Test;
+
+import static org.junit.Assert.*;
```

Comment:
tu mas import s *, ale asi je to jedno.

Reply:
Je to tak aj v `GamePhaseControllerTest` https://github.com/jbitta/pts24-java/blob/6221eaa3192f65742bc2114178acadb5517107e0/src/test/java/sk/uniba/fmph/dcs/game_phase_controller/GamePhaseControllerTest.java#L19

Ak je to v teste tak to asi nevadí, lebo to zjednodušuje ich písanie.